db/migrate/20170915095608_create_job_io_wrappers.rb

```diff
@@ -0,0 +1,15 @@
+class CreateJobIoWrappers < ActiveRecord::Migration[5.0]
```

Comment:
I didn't catch that this was in here before I submitted #255. I think we'll want to rebase this on that, so the migration has its own commit.

Reply:
Agree. Good idea.